src/app/address/controllers/address.js

```diff
@@ -1,12 +1,22 @@
 const BaseController = require("hmpo-form-wizard").Controller;
 
 class AddressController extends BaseController {
+  locals(req, res, callback) {
+    super.locals(req, res, (err, locals) => {
+      locals.searchValue = req.sessionModel.get("searchValue");
+      locals.addressLine1 = req.sessionModel.get("addressLine1");
+      locals.addressLine2 = req.sessionModel.get("addressLine2");
+      locals.addressTown = req.sessionModel.get("addressTown");
+      locals.addressPostcode = req.sessionModel.get("addressPostcode");
+      callback(null, locals);
+    });
+  }
+
   async saveValues(req, res, next) {
-    req.sessionModel.set("addressLine1", req.body["address-line-1"]);
-    req.sessionModel.set("addressLine2", req.body["address-line-2"]);
-    req.sessionModel.set("addressTown", req.body["address-town"]);
-    req.sessionModel.set("addressCounty", req.body["address-county"]);
-    req.sessionModel.set("addressPostcode", req.body["address-postcode"]);
+    req.sessionModel.set("addressLine1", req.body["addressLine1"]);
```

Comment:
Let's come back to this once we've got the flow built up more completely - it might turn out that we don't need this if we a) use the data from other screens b) use the `saveValues` callback pattern.